orted-mpir is a helper that avoids CFLAGS which can optimize out certain MPIR
debugging functionality. It is unversioned, leaks dependencies into other MPI
modules and libraries, and upstream agrees it shouldn't be installed.

https://github.com/open-mpi/ompi/pull/8892

diff --git orte/orted/orted-mpir/Makefile.am orte/orted/orted-mpir/Makefile.am
index 5c0dd33564..8a03fcd387 100644
--- orte/orted/orted-mpir/Makefile.am
+++ orte/orted/orted-mpir/Makefile.am
@@ -15,7 +15,7 @@
 
 CFLAGS = $(CFLAGS_WITHOUT_OPTFLAGS) $(DEBUGGER_CFLAGS)
 
-lib_LTLIBRARIES = lib@ORTE_LIB_PREFIX@open-orted-mpir.la
+noinst_LTLIBRARIES = lib@ORTE_LIB_PREFIX@open-orted-mpir.la
 lib@ORTE_LIB_PREFIX@open_orted_mpir_la_SOURCES = \
         orted_mpir_breakpoint.c \
         orted_mpir.h
